test/providers.test.js

```diff
@@ -21,6 +21,16 @@ test('json rpc fetch block info', withProvider(async (provider) => {
     expect(sameBlock.header.height).toEqual(height);
 }));
 
+test('json rpc fetch gas price', withProvider(async (provider) => {
+    const testHash = '8ehA3NYL5uSF8zefbnqnz66twYJ45rfst6SrqBNv7oka';
```

Comment:
Looks like this will break on the next hard-fork (already doesn't see to pass CI). Shouldn't hardcode a specific hash in tests.

Reply:
Let's use the status API to fetch the latest block hash and block height from it.